HugoPrinsloo/Cards
# ✨ Card naming

# Why 🤷🏼‍♂️ 

Added the ability to name cards

<img src="https://user-images.githubusercontent.com/12765774/42778426-514955c8-893d-11e8-9891-f2ba3dc8c44e.gif" width="300"/>


# How To Test 📲 
Add multiple cards and name. 
Make sure the card you name is the only card with the name
